example/lib/main.dart

```diff
@@ -46,8 +46,8 @@ class _MyAppState extends State<MyApp> {
     final String apptentiveKey;
     final String apptentiveSignature;
     if (Platform.isAndroid) {
-      apptentiveKey = "<YOUR_ANDROID_KEY>";
```

Comment:
Want to remove this!

Reply:
Sorry, should have commented on the right side of the diff, but there are FoxFax credentials in the diff.

Reply:
Thanks for catching that! I have updated the PR.